OpenRA.Game/FileFormats/Png.cs

```diff
@@ -231,22 +230,22 @@ static bool IsPaletted(byte bitDepth, PngColorType colorType)
 
 		public byte[] Save()
 		{
-			var pixelFormat = Palette != null ? PixelFormat.Format8bppIndexed : PixelFormat.Format32bppArgb;
+			var pixelFormat = Palette != null ? System.Drawing.Imaging.PixelFormat.Format8bppIndexed : System.Drawing.Imaging.PixelFormat.Format32bppArgb;
```

Comment:
I guess removal of System.Drawing here is for another PR...?

Reply:
Applies to all changes in this file ofcourse :D

Reply:
I still don't have the code or the understanding to write png directly, so this is deferred to the future.